config.toml.example

```diff
@@ -258,12 +258,24 @@
 [rust]
 
 # Whether or not to optimize the compiler and standard library.
-#
-# Note: the slowness of the non optimized compiler compiling itself usually
-#       outweighs the time gains in not doing optimizations, therefore a
-#       full bootstrap takes much more time with `optimize` set to false.
+# WARNING: Building with optimize = false is NOT SUPPORTED. Due to bootstrapping,
+# building without optimizations takes much longer than optimizing. Further, some platforms
+# (notably Windows) fail to build without this optimization (c.f. #65352).
```

Comment:
I think this should explicitly mention MSVC. There were no reports of this bug on windows-gnu and since it's using toolchain similar to Linux one it should be unaffected.

Reply:
I would sort of disagree, I think we should rather omit the mention on which platforms this may or may not fail. Seems like that's just fated to get further PRs trying to add/remove platforms over time.

Reply:
okay, I'll then remove the specific platform that fails.